src/abacus-ts/lib/resourceCacheManager.ts

```diff
@@ -55,9 +64,16 @@ export class ResourceCacheManager<T, U> {
       const delay = this.options.destroyDelayMs ?? 1000;
       entry.destroyTimeout = setTimeout(() => {
         const latestVal = this.cache[serializedKey];
-        if (!latestVal) return;
+        if (!latestVal) {
+          logAbacusTsError(
+            'ResourceCacheManager',
+            'unexpectedly couldnt find resource to destroy',
```

Comment:
unexpectedly couldnt find resource to destroy -> could not find resource to destroy